dkpro-core-opennlp-asl/src/scripts/build.xml

```diff
@@ -45,32 +45,32 @@
        <mkdir dir="target/download"/>
 
        <get 
-             src="https://bintray.com/inofficalbinarystore/generic/download_file?file_path=opennlp-af-afribooms.tar.gz" 
-             dest="target/download/opennlp-af-afribooms.gz" 
+             src="https://bintray.com/inofficalbinarystore/generic/download_file?file_path=opennlp-af-afribooms-perceptron.tar.gz" 
+             dest="target/download/opennlp-af-afribooms-perceptron.gz" 
              skipexisting="true"/>
        <untar 
-             src="target/download/opennlp-af-afribooms.gz" 
+             src="target/download/opennlp-af-afribooms-perceptron.gz" 
              dest="target/download/" 
              compression="gzip"/>
   
        <install-stub-and-upstream-file 
-          file="target/download/opennlp-af-afribooms/model-tagger-af.bin" 
+          file="target/download/opennlp-af-afribooms-perceptron/model-tagger-af.bin" 
           groupId="de.tudarmstadt.ukp.dkpro.core"
-          md5="cb9be0243fe7908d00a0749b760b81f5"
+          md5="491e782487531a9501e77e6b522d57d8"
           artifactIdBase="de.tudarmstadt.ukp.dkpro.core.opennlp"
-          upstreamVersion="20181017"
+          upstreamVersion="20181021"
           metaDataVersion="1"
           tool="tagger"
           language="af"
-          variant="afribooms"
+          variant="afribooms-perceptron"
           extension="bin">
             <metadata>
                <entry key="DC.rights" value="https://creativecommons.org/licenses/by-sa/4.0/"/>

```

Comment:
How about adding `DC.creator`, `DC.title`, and `DC.identifier`?

Reply:
As creator I would use DKPro Organization (see the follow up commits)

Reply:
The "DKPro Organization" (dkpro core developer team) is still an informal group. Formally, the code/documentation we produce is licensed to/by TU Darmstadt. I'd prefer if we continued not to include models in the kinds of things that we created - we only redistribute (some) models. That is one reason why models should be located in a third-party location and DKPro Core picks them up from there. So if you don't want to be the creator personally, maybe LangTech at UniDuis?